src/sidebar/components/annotation-publish-control.js

```diff
@@ -32,6 +47,12 @@ function AnnotationPublishControl({
   const setDefault = useStore(store => store.setDefault);
   const removeAnnotations = useStore(store => store.removeAnnotations);
 
+  if (!group) {
+    // If there is no group, don't render anything. A annotation does not need
+    // controls if it does not belong to a group.
```

Comment:
The reason `group` is null may be different than what is mentioned in this group. `annotation.group` may be a valid group ID, but it may not be loaded, in which case `store.getGroup(...)` will return undefined/null. In that case we can't tell if the group ID is valid or not.

Reply:
I'll make the comment change